README.md

```diff
@@ -14,6 +14,40 @@ See the section on plugins in the [ManageIQ Developer Setup](http://manageiq.org
 
 For quick local setup run `bin/setup`, which will clone the core ManageIQ repository under the *spec* directory and setup necessary config files. If you have already cloned it, you can run `bin/update` to bring the core ManageIQ code up to date.
 
+### Running Opentofu Runner locally
+First create the certs
+```
+mkdir certs
+openssl req -x509 -newkey rsa -days 1095 -keyout certs/tf_runner.key -out certs/tf_runner.crt -subj "/CN=server" -nodes -batch
+chmod 666 certs/tf_runner.crt certs/tf_runner.key
```

Comment:
Is 666 the correct permissions?  I'm surprised something doesn't complain about the permissions being too permissive.  It's dev setup but still it's curious.

Reply:
Yeah I didn't like this but haven't dug into why, but with 0640 opentofu-runner fails with a permissions error

Reply:
Okay `0644 ` works also which matches what we do on appliances, the issue is they run as a "stack" user in the container so we can't just do 0600 like I'd like to

Reply:
What about 644?  Hopefully, if it's considered "others" it only needs to read the file.

Reply:
lol, I didn't refresh and you already did it 🤣

Reply:
@jrafanie alternatively if I pass `--env NODE_ENV=development` I'm able to run this without SSL which would simplify things (no need to create certs), what do you think?

Reply:
haha, it's so much simpler now 👍